src/static/css/pages/offices.less

```diff
@@ -32,6 +32,17 @@
             });
         }
     }
+
+    .activity {
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/e446711d95276e18e0a3d778a6040713215dd5c3